### PR TITLE
UI followed screen

### PR DESCRIPTION
**Pull request by Keith-Go**
## Ticket
- [UI followed screen](https://trello.com/c/enbJxHBw/26-ui-followed-page)

## Video
[record.webm](https://github.com/luke-nguyen-goldenowl/join_me/assets/152830147/1593aaf7-4c03-4297-bcc2-0963a1fcb8c7)

### CODE REVIEW

File: lib/src/features/followed/upcoming/view/upcoming_event_followed_view.dart [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import 'package:flutter/material.dart';
+import 'package:myapp/src/features/followed/widget/ticket.dart';
 
 class UpcomingEventFollowedView extends StatelessWidget {
   const UpcomingEventFollowedView({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return const Placeholder();
+    return ListView.builder(
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/followed/upcoming/view/upcoming_event_followed_view.dart]: Convert into pagination

File: lib/src/features/followed/past/view/past_event_followed_view.dart [hunk 2]
```diff
@@ -1,10 +1,15 @@
 import 'package:flutter/material.dart';
+import 'package:myapp/src/features/followed/widget/ticket.dart';
 
 class PastEventFollowedView extends StatelessWidget {
   const PastEventFollowedView({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return const Placeholder();
+    return ListView.builder(
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/followed/past/view/past_event_followed_view.dart]: Convert into pagination

File: lib/src/features/followed/past/logic/past_bloc.dart [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:myapp/src/features/followed/past/logic/past_state.dart';
+import 'package:myapp/src/network/model/common/pagination/pagination.dart';
+
+class PastBloc extends Cubit<PastState> {
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/followed/past/logic/past_bloc.dart]: Cái file pagination_bloc.dart anh gửi em nó có set up sẵn cho pagination bloc rồi mà, sao ko dùng file đó mà tạo bloc bth rồi có mỗi hàm loadMore vậy em

**Comment by Keith-Go** [comment]
resolved

**Comment by luke-nguyen-goldenowl** [comment]
Use Utils for Date and Time, after fixing you can merge

**Pull request merged**